### PR TITLE
added details about the startup program

### PR DESCRIPTION
**Pull request by krishkatyal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,89 @@
...
+[![PyPI - Python
+Version](https://img.shields.io/badge/python-3.6%20%7C%203.7%20%7C%203.8%20%7C%203.9-blue)](https://pypi.org/project/nbox/)
+[![Downloads](https://pepy.tech/badge/nbox)](https://pepy.tech/project/nbox)
+![GitHub](https://img.shields.io/badge/license-Apache--2.0-blueviolet)
 
-# 🏖️ Nbox
+## 🧐 What is Nbox?
 
-`nbox` is NimbleBox.ai's official SDK.
+`nbox` provides first class support API for all NimbleBox.ai infrastructure (NBX-Build, Jobs, Deploy) and services (NBX-Workspaces) components. Write jobs using `nbox.Operators`
```
Comment (Aakash-kaushik) [inline README.md:9]: Maybe we can use First class api support instead

File: README.md [hunk 2]
```diff
@@ -1,39 +1,89 @@
...
 
-> The entire purpose of this package is to make using ML 🥶.
+# 🤷Why nimblebox
 
-```
-pip install nbox
-```
+- Write and execute code in Python
```
Comment (Aakash-kaushik) [inline README.md:13]: Make these catchy and quick.
`import notebooks from local machine`

**Review by Aakash-kaushik — COMMENTED** [review]
Looks good overall just use more straightforward sentences everywhere.

File: README.md (hunk 2 above)
Comment (krishkatyal) [reply README.md:13]: we can actually remove this part because I have added more catchy things below in features

File: README.md [hunk 3]
```diff
@@ -1,39 +1,89 @@
...
+
+
+### 🚀 Intuitive Dashboard
+![Deploy Landing Page](https://user-images.githubusercontent.com/89596037/181775468-cc342a30-d87e-4576-8bdd-8ffdd75ff759.gif)
+
+
+# 🏁 Get Started
+
+
+**Install the package from pipy:**
```
Comment (cshubhamrao) [inline README.md:49]: Typo: [**`PyPi`**](https://pypi.org/project/nbox)

File: README.md [hunk 4]
```diff
@@ -1,39 +1,89 @@
...
+
+```pip install nbox```
+
+
+For convinience you should add nbox to your path by setting up an alias. Throughout the rest of the documentation we will be using nbx as the CLI:
+
+```# go to your .bashrc or .zshrc and add
+alias nbx="python3 -m nbox"
 ```
```
Comment (cshubhamrao) [inline README.md:58]: No longer needed. `nbox >= 0.9.28` adds `nbox`, `nbx` and `nimblebox` as scripts

File: README.md [hunk 5]
```diff
@@ -1,39 +1,89 @@
...
+**Install the package from pypi:**
+
+```pip install nbox```
+
+
+For convinience you should add nbox to your path by setting up an alias. Throughout the rest of the documentation we will be using nbx as the CLI:
+
+```# go to your .bashrc or .zshrc and add
+alias nbx="python3 -m nimblebox "
 ```
```
Comment (cshubhamrao) [inline README.md:58]: Clarification: newer nbox (>= 0.9.28) add `nbx`, `nbox`, `nimblebox` as scripts– which means they are added as executables in the current (either venv or system python) environment's PATH, so setting the alias is no longer needed 😅

File: README.md [hunk 6]
```diff
@@ -69,6 +119,12 @@ pred = requests.post(
   header = {"Authorization": f"Bearer {key}"}
 ).json()
 ```
+# 🛟 How to get help?
+
+Join our [discord](https://discord.gg/qYZHxMaCsE) and someone from our community or engineering team will respond!
```
[the remark this answers is not in the document]
Comment (krishkatyal) [reply README.md:124]: what do you mean? it should only show you our discord invite link when clicked.

**Review by yashbonde — APPROVED** [review]

**Pull request merged**